src/commands/Social/marry.ts

```diff
@@ -24,36 +29,51 @@ async function askYesNo(channel: TextChannel | DMChannel, user: KlasaUser, quest
 		: YesNoAnswer.ImplicitNo;
 }
 
-export default class extends SkyraCommand {
-
-	public constructor(store: CommandStore, file: string[], directory: string) {
-		super(store, file, directory, {
-			cooldown: 30,
-			description: language => language.tget('COMMAND_MARRY_DESCRIPTION'),
-			extendedHelp: language => language.tget('COMMAND_MARRY_EXTENDED'),
-			runIn: ['text'],
-			usage: '(user:username)'
-		});
-
-		this.createCustomResolver('username', (arg, possible, msg) => {
-			if (!arg) return undefined;
-			return this.client.arguments.get('username')!.run(arg, possible, msg);
-		});
-	}
+@ApplyOptions<RichDisplayCommandOptions>({
+	cooldown: 30,
+	description: language => language.tget('COMMAND_MARRY_DESCRIPTION'),
+	extendedHelp: language => language.tget('COMMAND_MARRY_EXTENDED'),
+	runIn: ['text'],
+	usage: '(user:username)'
+})
+@CreateResolvers([
+	['username', (arg, possible, msg) => {
+		if (!arg) return undefined;
+		return msg.client.arguments.get('username')!.run(arg, possible, msg);
+	}]
+])
+export default class extends RichDisplayCommand {
 
 	public run(message: KlasaMessage, [user]: [KlasaUser | undefined]) {
-		return user ? this._marry(message, user) : this._display(message);
+		return user ? this.marry(message, user) : this.display(message);
 	}
 
-	public async _display(message: KlasaMessage) {
+	private async display(message: KlasaMessage) {
+		const response = await message.sendEmbed(new MessageEmbed()
+			.setDescription(message.language.tget('SYSTEM_LOADING'))
+			.setColor(BrandingColors.Secondary));
+
 		const users = message.author.settings.get(UserSettings.Marry);
 		if (users.length === 0) return message.sendLocale('COMMAND_MARRY_NOTTAKEN');
 
-		const usernames = await Promise.all(users.map(async user => `${await this.client.userTags.fetchUsername(user)} (${user})`));
-		return message.sendLocale('COMMAND_MARRY_WITH', [usernames]);
+		const usernames = chunk(await Promise.all(users.map(async user => {
+			const [userId] = await this.client.userTags.fetchEntry(user);
+			return `<@${userId}>`;
+		})), 20);
```

Comment:
Are you not going to use the username? It was added because sometimes mentions don't render well.
If not, change it to this:

```suggestion
		const usernames = chunk(users.map(user => `<@${user}>`), 20);
```

Reply:
in what cases would mentions not render well? Is this when they are married to someone they no longer share a server with?

Reply:
Yup. Or even when the married users are uncached on the user's side.

Reply:
Alright I'll revert this later today then. Currently updating pokemon API with DLC data.